libs/pages/logs/environment/src/lib/ui/sidebar-history/sidebar-history.tsx

```diff
@@ -42,32 +52,92 @@ export function SidebarHistory({ data, serviceId, versionId, pathLogs }: Sidebar
     },
   ]
 
-  const currentIndex = data?.findIndex((item) => item.id === versionId)
+  function findPositionById(data: DeploymentHistoryEnvironment[], versionId = '') {
+    const index = data.findIndex((item) => item.id === versionId)
+    return index !== -1 ? index : 0
+  }
+
+  function showNewTag(): boolean {
+    switch (environmentState) {
+      case StateEnum.DEPLOYING:
+      case StateEnum.DELETING:
+      case StateEnum.RESTARTING:
+      case StateEnum.BUILDING:
+      case StateEnum.STOP_QUEUED:
+      case StateEnum.CANCELING:
+      case StateEnum.QUEUED:
+      case StateEnum.DELETE_QUEUED:
+      case StateEnum.DEPLOYMENT_QUEUED:
+        return true
+      default:
+        return false
+    }
+  }
 
   if (data.length === 0) return null
 
+  const currentPosition = findPositionById(data, versionId)
+
   return (
-    <div className="flex justify-center border-b border-neutral-500 px-4 py-3">
-      <div>
+    <div className="flex border-b border-neutral-500 px-4 py-3">
+      <div className="flex">
+        <Button
+          dataTestId="btn-back-logs"
+          className="!border-r-0 !rounded-r-none"
+          style={ButtonStyle.DARK}
+          size={ButtonSize.TINY}
+          link={pathLogs}
+        >
+          <Icon name={IconAwesomeEnum.HOUSE} />
+        </Button>
         <Menu
           width={300}
           menus={menuHistory}
           arrowAlign={MenuAlign.CENTER}
-          onOpen={(isOpen) => setOpen(isOpen)}
           trigger={
-            <div
-              role="button"
-              className={`text-xs font-medium hover:text-brand-400 transition ${
-                open ? 'text-brand-400' : 'text-neutral-50'
-              }`}
+            <Button
+              className="!rounded-l-none w-[200px] mr-1.5"
+              style={ButtonStyle.DARK}
+              size={ButtonSize.TINY}
+              iconRight={IconAwesomeEnum.ANGLE_DOWN}
             >
-              <span className="inline-block mr-1">
-                Deployment - {dateFullFormat(data?.[currentIndex === -1 ? 0 : currentIndex]?.created_at)}
-              </span>
-              <Icon name={IconAwesomeEnum.ANGLE_DOWN} />
-            </div>
+              Deployment log history
+            </Button>
           }
         />
+        {showNewTag() && (
+          <Tooltip content="New deployment available!" side="right" intent="orange">
+            <div>
+              <Button
+                className="!text-orange-500 !border-orange-500 !bg-neutral-500 w-[50px]"
```

Comment:
Need to refacto `<Button />` to avoid !important for classes and Yael contacted to add it on the DS